tests/Makefile

```diff
@@ -9,12 +9,14 @@ UNIT_TEST_SRC += $(wildcard $(srcdir)/libmcount/*.c)
 UNIT_TEST_OBJ := $(patsubst %.c,%.ot,$(UNIT_TEST_SRC))
 UNIT_TEST_OBJ := $(filter-out %-nop.ot,$(UNIT_TEST_OBJ))
 
+WORKER = $(filter -j%, $(MAKEFLAGS))
```

Comment:
I suspect it also includes other flags after the `-j` on the command line, but it seems ok.  But I found that it messes up the screen with mixing output from the unit test and run test.

Reply:
But isn't it intended behaviour with mixed output? (Since each worker will run targets in parallel)

If you have no preference with it, then how about combining two target dependency into one?

```makefile
test: test_all

test_all:
$(QUIET_TEST)./unittest $(TESTARG)
$(QUIET_TEST)./runtest.py $(WORKER) $(TESTARG)
```

Reply:
That's ok too as long as it has dependency to the `unittest` binary.